wavenet/model.py

```diff
@@ -3,21 +3,36 @@
 from .ops import causal_conv, mu_law_encode
 
 
-def create_variable(name, shape):
+def _create_variable(name, shape):
```

Comment:
I thought the underscore was for member functions (methods) of classes. Am I wrong?